README.md

```diff
@@ -68,17 +68,10 @@ iex> Lens.get_and_map(data, lens, fn size -> {size, round(size)} end)
 
 ## Installation
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed as:
+[Available in Hex](https://hex.pm/packages/lens), the package can be installed as:
 
   1. Add lens to your list of dependencies in `mix.exs`:
 
         def deps do
           [{:lens, "~> 0.0.1"}]
         end
-
-  2. Ensure lens is started before your application:
```

Comment:
Actually I think we should leave this - tools like `exrm` use the application list to figure out which dependencies need to go into the release, so this is actually something you'll want to do in most cases.